pig/src/main/java/org/jboss/pnc/bacon/pig/impl/pnc/PncBuilder.java

```diff
@@ -82,10 +82,11 @@ private GroupBuild run(
             boolean tempBuildTS,
             RebuildMode rebuildMode) {
         log.info("Performing builds of build group {} in PNC", group.getId());
+        if (tempBuildTS)
+            log.info("Temporary builds with timestamp alignment are deprecated, running temporary build...");
```

Comment:
Shouldn't this be a warning ? :-) Does this mean its running without timestamp alignment and just running a temporary build?

Reply:
Yes it means exactly that, I guess it might be also warning but I believe we didn't receive feedback somebody is actually using it so I went with info. For deprecated, is no longer supported better?

Reply:
My understanding of deprecated is that e.g. "functionality x will be removed in a future version so migrate to the replacement y".
So yes, I agree and think  "Temporary builds with timestamp alignment is not supported ; running temporary build instead..." is  better.

In terms of info/warn ; personally I think if someone picks an option that you are trying to alert them that is no longer functions, warning is better. 

 @vibe13  - thought?